web/src/components/RunsTable/runsTable.js

```diff
@@ -159,13 +159,26 @@ class RunsTable extends Component {
             }
           }
 
+
           // Expand omniboard columns
           if ('omniboard' in data) {
             const omniboard = data['omniboard'];
             delete data['omniboard'];
             data = {...data, ...omniboard};
           }
 
+          // Add notes from comment if none has been saved in omniboard
+          if (!('notes' in data)) {
```

Comment:
```suggestion
          if (!('notes' in data && data['notes'])) {
```

Reply:
This is to ensure empty notes to get replaced with comment

Reply:
So the idea was to be able to completely delete a comment. My assumption is that the notes field in omniboard is only set when I make a change through omniboard. So when I delete the comment message, the notes field will be set but to an empty string. In your case, this would then be again replaced through the comment.
At least in my case, where I also sometimes put a comment by accident (when just repeating a command and changing only parts of it), this is not desirable and would need a workaround like putting space there to delete the comment.